barcode/writer.py

```diff
@@ -261,21 +264,23 @@ def _create_module(self, xpos, ypos, width, color):
         self._group.appendChild(element)
 
     def _create_text(self, xpos, ypos):
-        element = self._document.createElement('text')
-        attributes = dict(x=SIZE.format(xpos), y=SIZE.format(ypos),
-                          style='fill:{0};font-size:{1}pt;text-anchor:'
-                                'middle;'.format(self.foreground,
-                                                 self.font_size))
-        _set_attributes(element, **attributes)
         # check option to override self.text with self.human (barcode as
         # human readable data, can be used to print own formats)
         if self.human != '':
             barcodetext = self.human
         else:
-            barcodetext = self.text
-        text_element = self._document.createTextNode(barcodetext)
-        element.appendChild(text_element)
-        self._group.appendChild(element)
+            barcodetext = self.text		
+        for subtext in barcodetext.split('\n'):
+	        element = self._document.createElement('text')
+	        attributes = dict(x=SIZE.format(xpos), y=SIZE.format(ypos),
```

Comment:
Please indent to multiples of 4, rather than paren-based-indents:
```python
attributes = dict(
    x=SIZE.format(xpos),
    y=SIZE.format(ypos),
    style='fill:{0};font-size:{1}pt;text-anchor:middle;'.format(
        self.foreground,
        self.font_size,
    )
)
```

Might be preferable to use the `{}` literal, rather than `dict` here too.

Reply:
I kept the same style with "dict("  as in other parts of the code.

Reply:
I fixed the indent in commit  https://github.com/roth-a/python-barcode/commit/7c26d2f9084aeb31701e52fa99c9364e91223291